src/main/java/com/sintern/service/CompanyServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.sintern.service;
+
+import com.sintern.domain.Company;
+import com.sintern.repository.CompanyRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CompanyServiceImpl implements CompanyService {
+    private CompanyRepository companyRepository;
+    private EncryptionService encryptionService= new EncryptionServiceImpl();
```

Comment:
We use Spring for dependency injection. This is similar to the way we inject a Repository to a Service.

Reply:
Done